Frontend/CO.CDP.OrganisationApp.Tests/CustomScopeHandlerTests.cs

```diff
@@ -0,0 +1,88 @@
+using CO.CDP.OrganisationApp.Authorization;
+using CO.CDP.OrganisationApp.Constants;
+using Microsoft.AspNetCore.Authorization;
+using Moq;
+using FluentAssertions;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+
+namespace CO.CDP.OrganisationApp.Tests.Authorization;
+
+public class CustomScopeHandlerTests
```

Comment:
@andymantell @dharmverma Added a chunk of test scenarios. Thanks for your help guys.

Reply:
Nice 👍🏻